frontend/src/Judge/components/OtherMediaSubmission.js

```diff
@@ -1,5 +1,111 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Lightbox from 'react-images'
+import { Alert } from 'reactstrap'
 
-const OtherMediaSubmission = props => <div>Other Submission</div>
+import pdf from 'assets/pdf.svg'
+
+const PhotoContainer = styled.img`
+  height: 90%;
+  object-fit: scale-down;
+  width: 100%;
+`
+
+const PdfContainer = styled.img`
+  height: 100%;
+  object-fit: scale-down;
+  width: 45%;
+  background-color: #666666;
+  box-shadow: inset 2px 2px 1px rgba(0, 0, 0, 0.05);
+`
+
+const PdfText = styled.div`
+  position: absolute;
+  top: 80%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: black;
+`
+
+class OtherMediaSubmission extends Component {
+  static propTypes = {
+    path: PropTypes.string.isRequired
+  }
+
+  constructor (props) {
+    super(props)
+
+    this.state = {
+      lightboxIsOpen: false
+    }
+  }
+
+  closeLightbox = () => {
+    this.setState({ lightboxIsOpen: false })
+  }
+
+  openLightbox = () => {
+    this.setState({ lightboxIsOpen: true })
+  }
+
+  render () {
+    const { path } = this.props
+
+    return (
+      <Fragment>
+        {path.endsWith('.pdf') ? (
+          <div style={{ height: '80%', marginTop: '25px' }}>
+            <a href={`//localhost:3000/static/uploads/${path}`} target='_blank'>
+              <PdfContainer src={pdf} />
+              <PdfText>
+                <h3>Click to View</h3>
+              </PdfText>
+            </a>
+            <Alert
+              color='info'
+              style={{
+                marginTop: '45px',
+                width: '45%',
+                left: '50%',
+                transform: 'translate(-50%, -50%)'
+              }}
+            >
+              This submission may need to be viewed in person
```

Comment:
You might need some padding around this alert so it doesn't overlay, but this may be a needed thing for the voting panel.

Reply:
In our original sketches, we had this alert at the top of the screen.

Is it more noticeable at the top or the bottom?

Either way, agree w/ Tina it should be a block element and push down the Vote Panel as necessary. I can help w/ the CSS if necessary.